test/fixtures/identify-location.json

```diff
@@ -0,0 +1,49 @@
+{
+  "input": {
+    "type": "identify",
+    "userId": "user-id",
+    "timestamp": "2014",
+    "traits": {
+      "created": "2014-01-01",
+      "email": "jd@example.com",
+      "firstName": "john",
+      "lastName": "doe"
+    },
+    "context": {
+      "location": {
+        "city": "Dublin",
+        "country": "Ireland",
```

Comment:
In general it's a good idea to test not only the data you want and expect to receive, but also against the data you **don't** want to receive. Customers usually don't pass perfectly formatted data, so testing against the case where the user passes slightly unexpected data will make your code less brittle